cmd/root.go

```diff
@@ -110,38 +94,24 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
 	progress.PrimaryWorkflow = primaryWorkflow
 	progress.ProjectType = projectType
 
-	if cmd.Flags().Changed(cmdFlagKeyStack) {
-		progress.Stack = &cmdFlagStack
-	}
-	if progress.Stack == nil {
-		stack, err := phases.Stack(projectType)
-		if err != nil {
-			return err
-		}
-		progress.Stack = &stack
+	// stack
+	stack, err := phases.Stack(projectType)
+	if err != nil {
+		return err
 	}
+	progress.Stack = stack
 
-	if cmd.Flags().Changed(cmdFlagKeyAddWebhook) {
-		progress.AddWebhook = &cmdFlagAddWebhook
-	}
-	if progress.AddWebhook == nil {
-		wh, err := phases.AddWebhook()
-		if err != nil {
-			return err
-		}
-		progress.AddWebhook = &wh
+	// webhook
+	wh, err := phases.AddWebhook()
```

Comment:
I think this can be a flag also, a very straightforward boolean flag, wdyt?

Reply:
that is true, but for now i would like to focus on the requirements